src/main/res/values/strings.xml

```diff
@@ -138,8 +138,8 @@
     <string name="options_refresh_comments">Refresh Comments</string>
 
     <string name="options_past">Past Versions</string>
-    <string name="options_past_posts">Past Post List Versions</string>
-    <string name="options_past_comments">Past Comment List Versions</string>
+    <string name="options_past_posts">Past Versions of Post List</string>
+    <string name="options_past_comments">Past Versions of Comment List</string>
```

Comment:
Original was better

Reply:
The original was very unclear. It is a jumble of contextual words that all need to be taken into account to get any meaning out of it.